packages/design-system/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, forwardRef, HTMLAttributes, ReactElement, Ref } from 'react';
+
+import styles from './Breadcrumbs.module.scss';
+import Link from '../Link';
+import Dropdown from '../Dropdown/Dropdown';
+import { ButtonTertiary } from '../Button';
+import { StackHorizontal } from '../Stack';
+
+type BreadCrumbItems = {
+	label: string;
+	href: string;
+	target?: '_blank' | '_self';
+}[];
+
+type BreadCrumbProps = Omit<HTMLAttributes<HTMLElement>, 'className'> & {
+	as?: 'a' | ReactElement;
+	items: BreadCrumbItems;
+};
+
+const Breadcrumbs = forwardRef(
+	({ as = 'a', items, ...rest }: BreadCrumbProps, ref: Ref<HTMLElement>) => {
+		const maxItemLength = 4;
+
+		const buildEntries = useCallback(() => {
```

Comment:
What's the purpose of that `useCallback` here?

Reply:
Removed!